great_expectations_cloud/agent/actions/agent_action.py

```diff
@@ -15,6 +15,7 @@ class ActionResult(BaseModel):
     id: str
     type: str
     created_resources: Sequence[CreatedResource]
+    job_duration_milliseconds: Optional[float] = None  # noqa: UP007  # Python 3.8 doesn't support `X | Y` for type annotations
```

Comment:
This rule is ignored in the models.py file. I could also have moved this type over to that file, but it created a bunch of extra diffs and also makes the models.py file even larger. So I just added this comment instead; we can un-ignore it once we move away from older versions of Python

Reply:
There's a native `datetime.timedelta` type that better represents this concept, can we use that instead of `float`.
And `pydantic` already knows how to serialize and validate it.

https://docs.python.org/3/library/datetime.html#timedelta-objects

Reply:
Cool, thanks for the suggestion @Kilo59 . I updated this to be an optional timedelta instead.

For the logger.info mapping, though, I want to convert the timedelta to something easier to report on. Since we expect most jobs to take less than a minute, I went ahead and converted the timedelta to `total_seconds()` in the logs.